### ISSUE TITLE
Refactor Database deployment

### ISSUE THREAD

**Issue by Gretel5X** [flaviostutz/freepbx#26]
Is there a specific reason why the db is running alongside asterisk inside the container? I think it is common for Docker to separate concerns for a number of reasons?

**Comment by flaviostutz** [comment]
You are right. It would be better. I tried to do this before and ended up with a lot of issues because the way things were prepared regarding to dependencies, binaries etc so I gave up spending more time. But surely this would be great!

This is the best way of doing this.
